src/directives/infinite-scroll/infinite-scroll.component.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { InfiniteScrollModule } from './infinite-scroll.module';
+import { Component } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+
+@Component({
+    template: `<div class="row">
```

Comment:
I think we can probably simplify this template to just:

```html
<div [uxInfiniteScroll]="load" [filter]="filterText" [pageSize]="pageSize" [loadOnScroll]="loadOnScroll"></div>
```

I don't think we need the input or anything as we are just updating the `filterText` variable in code